Backends/CLX/input.lisp

```diff
@@ -397,31 +397,6 @@
 (defmethod (setf port-frame-keyboard-input-focus) (focus (port clx-basic-port) frame)
   (setf (frame-properties frame 'focus) focus))
 
-;; FIXME: What happens when CLIM code calls tracking-pointer recursively?
-;; I expect the xlib:grab-pointer call will fail, and so the call to
-;; xlib:ungrab-pointer will ungrab prematurely.
-
-;;; XXX Locks around pointer-grab-sheet!!!
-
-(defmethod port-grab-pointer ((port clx-basic-port) pointer sheet)
```

Comment:
dragging between frames works just fine in this branch. try opening two dnd-commented frames and drag between them to see.

as of grabbing, there is relevant discussion on #clim channel:
https://irclog.tymoon.eu/freenode/%23clim?around=1565353219#1565353219

I'll leave the port interface but won't call it in tracking-pointer. If there is a usecase which breaks because of lack of pointer grabbing then we should grab it inside drag-output-record (not tracking pointer).

Reply:
OK.